python/doc/examples/probabilistic_modeling/distributions/plot_create_draw_multivariate_distributions.py

```diff
@@ -15,16 +15,16 @@
 # Create a multivariate model with `JointDistribution`
 # -------------------------------------------------------
 #
-# In this paragraph we use :math:`~openturns.JointDistribution` class to
+# In this paragraph we use :class:`~openturns.JointDistribution` class to
 # build multidimensional distribution described by its marginal distributions and optionally its dependence structure (a particular copula).
 #
 
 
 # %%
 # We first create the marginals of the distribution :
 #
-#   - a Normal distribution ;
-#   - a Gumbel distribution.
+#   - a :class:`~openturns.Normal` distribution ;
+#   - a :class:`~openturns.Gumbel` distribution.
 #
 marginals = [ot.Normal(), ot.Gumbel()]
```

Comment:
```suggestion

marginals = [ot.Normal(), ot.Gumbel()]
```
I am trying to remove the large gap between the text and the code line:
![image](https://github.com/openturns/openturns/assets/31989332/2d9c8250-cce9-4217-9692-273ab1877a21)